### ISSUE TITLE
Elementor 2.9.8 plugin causing mixed content to be loaded (all images HTTP, rest HTTPS)

### ISSUE THREAD

**Issue by MsGyver** [elementor/elementor#11392]
[system-info-beanerdgirl.club-18-05-2020.txt](https://github.com/elementor/elementor/files/4643911/system-info-beanerdgirl.club-18-05-2020.txt)
Hello,

On one of my websites, one of two WordPress sites I run Elementor on (combined with Rife Free & imported template), all the images getting called are loaded as HTTP, rather than HTTPS, causing the browser to flag this as unsecure.

Only when the Elementor plugin (I run 2.9.8) is disabled, does the webiste display in full HTTPS, but obviously loses all of the Elementor styling.

Already verified:
* https://www.whynopadlock.com/results/dfbe3b22-28c5-4b7e-8abe-24a62e30c9e5
* all images removed, and verified they were re-uploaded whilst being logged in to WordPress in HTTPS environment
* using Better Search Replace, I found and fixed 43 instances where HTTP was being called rather than HTTPS
* my SSL certitifcate has been re-issued.
* whynopadlock still flags the same images as being loaded HTTP

The only 3 plugins I have installed and use are:
Apollo13 Framework Extensions 1.8.4
Elementor 2.9.8
Rife Elementor Extensions & Templates 1.1.3

Only 1 theme installed:
Rife Free 2.4.8 - using free import template "Raven")

Is this something you have encountered before?

Best regards,

Laura

**Comment by shilo-ey** [comment]
Hi @MsGyver 
Thanks for your input.

Please go into your page and try to locate the image widgets with the mixed content paths.

Once found (you can use the Navigator) to do so - simply select the image again.

As far as I can see - there's no such files in your site and that's probably why you are getting this error.

http://www.beanerdgirl.club/wp-content/uploads/2018/06/bang.png

for example - leads to 404 even when accessed via `https://` prefix

Thanks

**Comment by MsGyver** [comment]
Thank you for your evaluation, it is wasn't correct though, but it pointed me in the right direction.
The 9 other images on my site loaded in both http:// and https://

I removed all of them, removed the "story using year/month" setting, uploaded all again, and search replaced all links to images

It works again.

**Comment by MsGyver** [comment]
Weird that I've had to do that, though. The other wbsite I use Elementor & Rife Free on, that is much more intricate, with more moving parts, and a multitude in images, was, and is, fully secure.

Would you know how this error is created, so that I can prevent this from happenig again?

**Comment by shilo-ey** [comment]
I don't really understand what is "story using year/month" option.

Perhaps it's something related to your theme or extension that doesn't fully comply with Elementor guidelines.

**Comment by MsGyver** [comment]
That was a type, I meant to write store, instead of story.
![image](https://user-images.githubusercontent.com/65543415/82244753-fe80cf80-9941-11ea-82f7-5aa48080a6f4.png)
super basic WordPress stuff, on how to store images. It defaults to year/month, which makes it easy for blog style sites to work nicely. For me that didn't apply, so I removed it; It wasn't part of the problem, but I did clean it up nevertheless
